openupgradelib/openupgrade.py

```diff
@@ -1031,13 +1032,16 @@ def migrate(no_version=False, use_env=None, uid=None, context=None):
     logging purposes.
     """
     def wrap(func):
-        def wrapped_function(cr, version):
+        @wraps(func)
+        def wrapped_function(cr, version, no_version=no_version, use_env=use_env,
+                             uid=uid, context=context):
```

Comment:
I didn't ask if it breaks anything, it obviously doesn't. The question is: What do you expect from returning a function that takes parameters which will never be passed and always fall back to their default value, which you set from names available in the closure anyways?